misk/src/main/kotlin/misk/MiskServiceModule.kt

```diff
@@ -108,7 +108,27 @@ class MiskCommonServiceModule : KAbstractModule() {
 
     val builder = ServiceGraphBuilder()
 
+    // Avoid adding services to the builder twice, while we migrate the API.
```

Comment:
Why? I think it should explode if it’s added 2x

Reply:
Side effect of supporting both the `DependentService` and `ServiceGraphBuilder` APIs at the same time! `SGBuilder` uses `ServiceEntry` which grabs the `Service` instance from the injector, but the `DependentService` API grabs a set of all `Services` bound ever! Since `install(ServiceModule<>())` creates a binding, you'll get a service twice :(

Reply:
I had to add this change because `JettyService` would cause everything to explode even though it *is* only registered once.